src/lint/collect-spelling-diagnostics.ts

```diff
@@ -51,6 +51,10 @@ let matchers = [
     pattern: /\r/gu,
     message: 'Only Unix-style (LF) linebreaks are allowed',
   },
+  {
+    pattern: /(?<=\b[Ss]teps? )\d/gu,
```

Comment:
How does this not trigger all over [Algorithm Conventions](https://tc39.es/ecma262/#sec-algorithm-conventions) where we describe how to read algorithm steps?